docs/source/install/external_modules.rst

```diff
@@ -2,21 +2,63 @@ External modules
 ================
 
 MALA can be coupled to external libraries for data pre- and postprocesing.
-The DFT code QuantumESPRESSO is used for the calculation of the density
+The DFT code Quantum ESPRESSO is used for the calculation of the density
 contributions to the total energy, while the LAMMPS code is used to calculate
 SNAP descriptors on the real space grid of a simulation cell, either for
 training or inference.
 
 Quantum ESPRESSO (total energy module)
 ***************************************
 
-The total energy module has been moved to its own fork of QE.
-You can find it at
-https://gitlab.com/casus/q-e/-/tree/tem_original_development after
-gaining access from the MALA development team. After checking out this
-fork and branch, go to ``total_energy_module/`` and follow the installation
-instructions described therein.
+To run the total energy module, you need a full Quantum ESPRESSO (version
+6.4.1.) and install the python bindings for it.
 
+Build Quantum ESPRESSO
+-----------------------
+
+* Clone QE source code: ``git@gitlab.com:QEF/q-e.git``
+* Chage into the ``q-e`` directory and check out the correct branch
+  (the total energy module is based on version 6.4.1): ``git checkout qe-6.4.1``
+* Make sure you have an (MPI-aware) F90 compiler such as ``mpif90`` (e.g.
+  Debian-ish machine: ``apt install openmpi-bin``, on an HPC cluster something
+  like ``module load openmpi gcc``). In fact the extension module that we build
+  doesn't use MPI at the moment, so a non-MPI compiler should also work (e.g.
+  ``gfortran`` or ``ifort``). In any case, make sure to use the same compiler
```

Comment:
These are the instructions we wrote back in the day. I'm not up to date re. the build process but just to make sure: is it still true that the extension module is serial?

Reply:
You're right, this is outdated. We now build in parallel. I have fixed this line.